src/main/java/seedu/address/logic/parser/SortMeetingParser.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATETIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_LOCATION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MEETING_TITLE;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import seedu.address.logic.commands.AutocompleteResult;
+import seedu.address.logic.commands.SortMeetingCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new AddMeetingCommand object.
+ */
+public class SortMeetingParser implements Parser<SortMeetingCommand> {
+    /**
+     * Returns true if none of the prefixes contains empty {@code Optional} values in the given
+     * {@code ArgumentMultimap}.
+     */
+    private static boolean arePrefixesValidUnique(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
+        if (argumentMultimap.getLength() != 2) {
+            return false;
+        } else {
+            return Stream.of(prefixes).anyMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
+        }
+    }
+
+    @Override
+    public AutocompleteResult getAutocompleteSuggestion(String args) {
+        return new AutocompleteResult(null, false);
+    }
```

Comment:
How about having autocompletion that allows the user to `TAB` and cycle through the available `Prefix` to sort by? something like this: 
```suggestion
@Override
    public AutocompleteResult getAutocompleteSuggestion(String args) {
        String[] argsSplit = args.trim().split(" ");
        if (args.isEmpty()) {
            return new AutocompleteResult(PREFIXES[0], false);
        }
        String lastPrefix = argsSplit[argsSplit.length - 1];

        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(" " + lastPrefix, PREFIXES);
        boolean returnNext = false;

        for (Prefix p : PREFIXES) {
            if (returnNext) {
                return new AutocompleteResult(p, true);
            }

            if (argMultimap.getValue(p).isPresent() && p.toString().equals(lastPrefix)) {
                returnNext = true;
            }
        }

        if (!args.endsWith("/")) {
            return new AutocompleteResult(null, false);
        }

        return new AutocompleteResult(PREFIXES[0], true);
    }
```